src/api/app/validators/workflow_filters_validator.rb

```diff
@@ -1,11 +1,19 @@
 class WorkflowFiltersValidator < ActiveModel::Validator
   SUPPORTED_FILTER_VALUES = [:only, :ignore].freeze
+  DOCUMENTATION_LINK = 'https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html' \
+                       '#sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters'.freeze
```

Comment:
Do you think we can break this link into two parts? For example 
```
DOCUMENTATION_BASE_LINK = 'https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html'
```
And then define the link path:
```
DOCUMENTATION_PATH = '#sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters'
```
Reason is `DOCUMENTATION_BASE_LINK` is used in few places above and it's common. We don't have to hard code the `DOCUMENTATION_BASE_LINK` in those places.

And do you think we can move `DOCUMENTATION_BASE_LINK`  and `DOCUMENTATION_PATH` to `src/api/app/models/token/workflow.rb` ?

Reply:
I prefer to keep this simple. For now, we only have twice a link to the documentation. If this grows in the future, we can change it.

Reply:
I changed my mind, I'll do it.